Libraries/VtkVgCore/vtkVgTrack.h

```diff
@@ -257,13 +258,21 @@ class VTKVG_CORE_EXPORT vtkVgTrack : public vtkObject
   vtkSetMacro(Type, int);
   vtkGetMacro(Type, int);
 
-  // Description:
-  // Set/Get the PVO classification for this track.  These must sum to 1;
-  // to guarantee summing to 1, we normalize on input.
+  // DEPRECATED (TODO: remove this)
   void SetPVO(double person, double vehicle, double other);
   void SetPVO(double pvo[3]);
   vtkGetVector3Macro(PVO, double);
```

Comment:
I'm not sure how that would be valuable. I'm not sure if it's true as of this PR (if not, it will be true as of the one that will follow), but right now we have WAMI Viewer using TOC exclusively, and vsPlay/viqui doing... "something else", but at the vtkVgTrack level anyway, being totally unaware of TOCs. Treating them as separate avoids making any decisions how to convert between the two.